tests/unit/.eslintrc

```diff
@@ -0,0 +1,22 @@
+{
+  'extends': '../../.eslintrc',
+
+  'env': {
+    'mocha': true
+  },
+
+  'globals': {
+    'expect': false,
+    'sandbox': false,
+    'document': false,
+    'window': false,
+    'React': false,
+    'ReactDOM': false,
```

Comment:
Why hide dependencies? If you need React in your module, then make it explicit. Implicit dependencies are never a good thing. Think about what happens when you move your module to an environment that doesn't expose React as a global. It's this kind of unexpected things that make maintaining code harder.